### DISCUSSION TITLE
`modular install mojo` should automatically add mojo in PATH

### DISCUSSION THREAD

**Discussion by remorses** [modular/modular#2893] [Ideas]
`modular install mojo` should automatically add mojo in PATH, there is no reason to let the user do it themselves, a lot of users on Mac also don't even know if they are using zsh or bash

**Comment by ksandvik** [comment]
This means that the installer should be smart and figure out how to edit various PATH settings depending on shell used. 

Most developers know how or should know to edit PATH statements. Maybe another option is have a fixed location for all Mojo tools binaries (similar to other languages).
